src/webrtc/call.ts

```diff
@@ -1569,13 +1569,20 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
 
         try {
             await this.peerConn.setLocalDescription(answer);
+
+            // make sure we're still going
+            if (this.callHasEnded()) return;
+
```

Comment:
I wonder if this should be in `setState()`

Reply:
I should go sleep, anyway...

Reply:
Yeah, I did wonder about this and it seems quite elegant to basically just disallow state transitions out of 'ended', at the price of having to wrap all our setState calls in a try/catch (or some way to make the caller stop what it's doing).

Reply:
Yeah, so what I meant by _I should go to sleep, anyway..._ is that there is no simple way to do this, so I think this is actually fine (same for the other comment)